ui/admin/form.php

```diff
@@ -15,7 +15,7 @@
 $groups = PodsInit::$meta->groups_get( $pod->pod_data['type'], $pod->pod_data['name'], $fields );
 
 $pod_options = $pod->pod_data['options'];
-$pod_options = apply_filters( 'pods_advanced_content_type_pod_data_' . $pod->pod_data['name'], $pod_options, $pod->pod_data['name'] );
+$pod_options = apply_filters( "pods_advanced_content_type_pod_data_{$pod->pod_data['name']}", $pod_options, $pod->pod_data['name'] );
```

Comment:
Doesn't fit the same way the others were changed, can you set as var and then put in filter name?

Reply:
Addressed in a8f4ee864afea764b8ba67c836dc1c675c1f5582.